docs/team/jerrrren.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: John Doe's Project Portfolio Page
```

Comment:
Update to 'Jer En's Project Portfolio Page'

Reply:
I have rectified this